index.js

```diff
@@ -17,15 +17,12 @@ export class VersionNotFoundError extends Error {
 	}
 }
 
-export default async function packageJson(packageName, options) {
-	options = {
-		version: 'latest',
-		omitDeprecated: true,
-		...options,
-	};
+export default async function packageJson(packageName, options = {}) {
+	options.version ??= 'latest';
+	options.omitDeprecated ??= true;
```

Comment:
You are mutating the input object, which is a no-no.

A better solution is destructuring with default values.